elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -108,6 +108,8 @@ RUN for plugin_path in `cat /tmp/plugins/plugins.list`; do \
       elasticsearch-plugin install --batch file:/tmp/plugins/$plugin_path; \
     done
 
+# Pull performance analyzer agent(RCA) from the plugin directory out into ESHOME
+RUN if [ -d plugins/opendistro_performance_analyzer/performance-analyzer-rca ]; then mv plugins/opendistro_performance_analyzer/performance-analyzer-rca ./; fi
```

Comment:
why do we need this "move"?  could you use the same way as PA does?

Reply:
The move is done by post-install hooks that DEB and RPM packages expose. There is no equivalent in docker world(where we install the plugin using zip) but to do that manually.